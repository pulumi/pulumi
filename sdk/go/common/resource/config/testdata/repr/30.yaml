my:outer:
  inner:
    nested:
      secure: securevalue
