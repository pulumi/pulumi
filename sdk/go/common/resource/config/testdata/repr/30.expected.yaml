decrypt:
  my:outer: '{"inner":{"nested":{"secure":"securevalue"}}}'
paths:
  my:outer:
    value:
      inner:
        nested:
          secure: securevalue
    string: '{"inner":{"nested":{"secure":"securevalue"}}}'
    redacted: '{"inner":{"nested":"[secret]"}}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABp/4JmAAEFaW5uZXIXbWFwW3N0cmluZ11pbnRlcmZhY2Uge33/gkMAAQZuZXN0ZWQXbWFwW3N0cmluZ11pbnRlcmZhY2Uge33/gh8AAQZzZWN1cmUGc3RyaW5nDA0AC3NlY3VyZXZhbHVl
    secure: true
    isObject: true
    secureValues:
    - securevalue
  my:outer.inner:
    value:
      nested:
        secure: securevalue
    string: '{"nested":{"secure":"securevalue"}}'
    redacted: '{"nested":"[secret]"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABG/4JDAAEGbmVzdGVkF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4IfAAEGc2VjdXJlBnN0cmluZwwNAAtzZWN1cmV2YWx1ZQ==
    secure: true
    isObject: true
    secureValues:
    - securevalue
  my:outer.inner.nested:
    value:
      secure: securevalue
    string: securevalue
    redacted: '[secret]'
    object: GBAABnN0cmluZwwNAAtzZWN1cmV2YWx1ZQ==
    secure: true
    isObject: false
    secureValues:
    - securevalue
